1-exercises/C-highlight-words/script.js

```diff
@@ -1,5 +1,28 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  const PARAGRAPH = document.createElement("p"),
+    SELECT = document.createElement("select"),
+    WORDS = paragraph.split(" ");
```

Comment:
No need for an upper casing here, as it's not an enumerator or static property.

Reply:
oh I thought the general rule of thumb was to always use a const if you don't plan on reassigning the value and if you do use a const you should use all caps. I'll google what an enumerator and a static property is.

Reply:
Every style guide has different recommendations on casing, see for example the Airbnb style guide on casing here: https://github.com/airbnb/javascript#naming--uppercase

Typically, you will only want uppercase casing when a variable is constant and also static i.e. it won't change in the different runs of your app.